src/components/Experiences/ExperienceCards.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Icon from "../Common/Icon";
+
+export default function ExperiencesCards() {
+    return (
+        <div className="w-48  mx-2 flex-shrink-0 relative border border-gray-300 shadow-lg mb-4 rounded-md">
+            <div className="w-full  relative pb-3/4">
+                <img
+                    src="https://ik.imagekit.io/q5edmtudmz/post1_fOFO9VDzENE.jpg"
```

Comment:
Convert it to an asset.

Reply:
As this is a known issue - https://github.com/Kites-Foundation/hello-munnar-frontend/issues/36 resolving it for now.